alkira/resource_alkira_connector_gcp_interconnect_helper.go

```diff
@@ -29,84 +28,49 @@ func expandGcpInterconnectCustomerGateways(cg []interface{}) ([]alkira.Connector
 	return customerGateways, nil
 }
 
-func expandGcpInterconnectSegmentOptions(so []interface{}, instanceName string, m interface{}) ([]alkira.ConnectorGcpInterconnectSegmentOption, error) {
-	if so == nil || len(so) == 0 {
+func expandGcpInterconnectSegmentOptions(sO []interface{}, m interface{}) ([]alkira.ConnectorGcpInterconnectSegmentOption, error) {
```

Comment:
so instead of s0

Reply:
I changed it form so to sO because I wanted it to be camel case. `segmentOptions` shortened to `sO` as it is customary to use camelCase and shorthands in golang.

Reply:
could we actually have a meaningful name for this var? `m` and `d` are commonly used in TF plugin code. `sO` or `so` is very confusing.